src/components/LightUpChinatown/LightUpChinatownPage.tsx

```diff
@@ -133,7 +153,48 @@ const LightUpChinatownPage = () => {
     </React.Fragment>
   );
 };
+// left: 42%;
+// top: 70%;
+const Button = styled.div`
+  line-height: 40px;
+  width: 15%;
```

Comment:
We should be using a fixed height and width here and centering it in order to make it sit on the page correctly